packages/lndr/pending-transaction/index.ts

```diff
@@ -6,12 +6,14 @@ export default class PendingTransaction {
   creditRecord: CreditRecord
   submitter: string
   hash: string
+  multiHashes?: string[]
```

Comment:
What is multiHashes? Should it have a more descriptive name? Is it used somewhere?